src/github.com/getlantern/checkfallbacks/checkfallbacks.go

```diff
@@ -149,6 +150,7 @@ func testFallbackServer(fb *client.ChainedServerInfo, workerId int) (output full
 		output.info = []string{"\n" + string(reqStr)}
 	}
 
+	req.Header.Set("X-LANTERN-AUTH-TOKEN", fb.AuthToken)
```

Comment:
I saw it [here](https://github.com/getlantern/lantern/blob/valencia/src/github.com/getlantern/flashlight/client/chained.go#L98), but that's right, is application code. It's also [here](https://github.com/getlantern/lantern/blob/valencia/src/github.com/getlantern/flashlight/config/config.go#L578) and [here](https://github.com/getlantern/lantern/blob/valencia/src/github.com/getlantern/flashlight/client/reverseproxy.go#L33), so from the library design point of view, it is expected to be defined always in the user code.